pkg/ddc/juicefs/transform.go

```diff
@@ -176,16 +176,17 @@ func (j *JuiceFSEngine) allocatePorts(dataset *datav1alpha1.Dataset, runtime *da
 	// if not use hostnetwork then use default port
 	// use hostnetwork to choose port from port allocator
 
-	expectedPortNum := 2
+	exportWorkerPodNum := 1
```

Comment:
Is `export` a typo here? Did you mean `expected`?

Reply:
yeah.